src/Api/AbstractCore.php

```diff
@@ -325,6 +330,7 @@ public function login(?string $code = null) : array
                     'authCode'    => $auth_code,
                     'redirectUrl' => 'nucleus:rest',
                 ],
+                //"ds" => "3acf525ee688016f4a8e5e5ebe307a0f4bf400e6b33e34d0ecc057358820c9ac/cc39"
```

Comment:
This seems to be something new, and feels like without that we get back 403 for that request.

Reply:
Well done, but unfortunately I do not know how it works myself. Without that you can't log in?

Reply:
Without this param doesn't matter. You got 403 because the auth_code need get again before this request.